WordPress/src/main/res/color/secondary_gray_20_disabled_selector.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:alpha="@dimen/disabled_alpha" android:color="?attr/colorOnSurface" android:state_enabled="false" android:state_selected="false" />
+    <item android:alpha="@dimen/disabled_alpha" android:color="?attr/colorOnSurface" android:state_enabled="false" android:state_selected="true" />
+    <item android:color="@color/gray_20" android:state_enabled="true" android:state_selected="true" />
```

Comment:
What do you think about using the material color attribute here, like wpColorOnSurfaceMedium? Looks very similar :)

Reply:
ahah 😄 . It was the only state not using material attr 🤦 ! Good catch 👍 . Changed in fcf46ac15fe24b4c8de9facb92a7019aace15900